extern/ProtoGeometry/libg_locale/en_US/LC_MESSAGES/libg.po

```diff
@@ -252,14 +252,14 @@ msgid ""
 msgstr ""
 
 #: LibGCore/src/ds/ds_arc.cpp:342
-msgid "Cannot create Arc: base object NULL"
+msgid "Cannot create an Arc with the given input points."
```

Comment:
@aparajit-pratap should we bring this in? Looks like there is an actual change here and below.